configure.ac

```diff
@@ -1,12 +1,33 @@
 AC_PREREQ([2.61])
 AC_INIT([mstpd], [0.0.4], [https://github.com/mstpd/mstpd])
 
+# bridge-stp must be installed in /sbin/, and the associated scripts and
+# executables should also be installed outside of /usr/.  Set the default
+# exec_prefix so they are installed in the correct places by default.
+# AC_PREFIX_DEFAULT(...) can be used to override the default prefix, but there
+# doesn't seem to be any standard way to override the default exec_prefix, so
+# just do it manually.
+test "x$exec_prefix" = xNONE && exec_prefix=''
```

Comment:
so, it doesn't work to call `./configure --exec-prefix= --prefix=` ?
I always thought that was enough to set the prefix values to whatever was needed

if this is needed for the RPM/DEB creation, I'd move the arguments there, and not override them here